packages/editor-ui/src/utils/sortUtils.ts

```diff
@@ -240,6 +246,23 @@ export function sublimeSearch<T extends object>(
 				accu: null | { matched: boolean; outScore: number },
 				{ value, weight }: { value: string; weight: number },
 			) => {
+				// Check for whole word match first
+				if (containsWord(value, filter)) {
+					return {
+						matched: true,
+						outScore: 1000 * weight, // High score for whole word matches
+					};
+				}
+
+				// Check for substring match (case-insensitive)
+				if (value.toLowerCase().includes(filter.toLowerCase())) {
```

Comment:
why is this needed too?

Reply:
I think this might already be addressed by the fuzzy search algorithm

Reply:
Indeed, it was a leftover.